ui/hooks/useDisplayName.ts

```diff
@@ -32,6 +33,7 @@ export function useDisplayNames(
   const erc20Tokens = useERC20Tokens(requests);
   const watchedNFTNames = useWatchedNFTNames(requests);
   const nfts = useNFTs(requests);
+  const domainResolutions = useSelector(getDomainResolutions);
```

Comment:
To clarify, this is only for when the user has already entered a domain name somewhere that was resolved to an address and then cached?

Reply:
Yes, I believe this is the main use case we want to support, because it came up during the design review that a user might be confused by searching for an ens address during the send flow, and then not have that translated in the petnames component.

One other advantage of this scope is that there is no additional loading state that needs to be managed/designed.